### DISCUSSION TITLE
Tizen 2.4.0 Support?

### DISCUSSION THREAD

**Discussion by SxckMySyp** [jhassan8/crunchyroll-tizen#74] [Q&A]
Is Tizen 2.4.0 working/supported for this because somewhere on Youtube someone said it will not work on version of Tizen which is below 3.0.
Ty for your answers (:

**Comment by jhassan8** [comment]
Hi @SxckMySyp, will you have the exact firmware version, for example, my tvs has version 2501, 1310 and the app works correctly.

Maybe the following link will help you [https://github.com/jhassan8/crunchyroll-tizen/discussions/70](https://github.com/jhassan8/crunchyroll-tizen/discussions/70)
